manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Drumstick.dmidiplayer
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Drumstick Projects
+PublisherUrl: https://dmidiplayer.sourceforge.io
+PublisherSupportUrl: https://sourceforge.net/p/dmidiplayer/tickets/
+# PrivacyUrl: 
+Author: Pedro López-Cabanillas <plcl@users.sourceforge.net>
+PackageName: Drumstick Multiplatform MIDI File Player
```

```suggestion
PackageName: dmidiplayer
```
This must match Control Panel.

---

OK